app/assets/stylesheets/articles.scss

```diff
@@ -1107,7 +1107,10 @@
             position: absolute;
             color: $black;
             right: 5px;
-            top: 2px;
+            top: 50%;
+            -webkit-transform: translateY(-50%);
+            -ms-transform: translateY(-50%);
```

Comment:
Hi @phongtruongg, I tested this with Firefox and it works well. One question: I don't think you need `-webkit-transform`, `transform` is supported by all browsers: https://caniuse.com/#feat=transforms2d - https://developer.mozilla.org/en-US/docs/Web/CSS/transform

Same goes for `-ms-transform`, it dates back to IE9 :D

Reply:
I've removed it